extensions/healthcheck/src/main/java/org/apache/unomi/healthcheck/servlet/HealthCheckServlet.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.unomi.healthcheck.servlet;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.unomi.healthcheck.HealthCheckService;
+import org.osgi.service.http.HttpContext;
+import org.osgi.service.useradmin.Authorization;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * A simple servlet that provides a health check endpoint.
+ */
+public class HealthCheckServlet extends HttpServlet {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(HealthCheckServlet.class.getName());
+
+    private final HealthCheckService service;
+    private final ObjectMapper mapper;
+
+    public HealthCheckServlet(HealthCheckService service) {
+        LOGGER.info("Building healthcheck servlet...");
+        this.service = service;
+        mapper = new ObjectMapper();
```

Comment:
There is the class CustomObjectMapper which provides the object mapper which is used in UNOMI, can we use it instead here?
https://github.com/apache/unomi/blob/a44710fac7fb6f6bc69f24e65e309bc7b5c0871b/persistence-spi/src/main/java/org/apache/unomi/persistence/spi/CustomObjectMapper.java#L46

Reply:
That mapper has very specific features configured so I'm not sure it fits with the healthcheck
More than that, it is not available at karaf startup without unomi started and adding that kind of dependency is not  good choice for me.